mne/tests/test_epochs.py

```diff
@@ -1901,7 +1901,11 @@ def make_epochs(picks, proj):
     epochs_meg2 = epochs_meg.copy()
     epochs_meg2.info['chs'][1]['ch_name'] = epochs_meg2.info['ch_names'][0]
     epochs_meg2.info._update_redundant()
-    assert_raises(RuntimeError, add_channels_epochs, [epochs_meg2, epochs_eeg])
+    with warnings.catch_warnings(record=True) as w:
```

Comment:
It helps to add a few-word comment with each catch saying what it's catching, makes it easier to remove them later if we fix whatever was causing the warning